### PR TITLE
chore(cli-repl): cleanup CLI argument documentation MONGOSH-571

### PR DESCRIPTION
**Pull request by rose-m**
New help:
```
  $ mongosh [options] [db address]

  Options:

    -h, --help                                 Show this usage information
        --host [arg]                           Server to connect to
        --port [arg]                           Port to connect to
        --version                              Show version information
        --nodb                                 Don't connect to mongod on startup - no 'db address' [arg] expected
        --retryWrites                          Automatically retry write operations upon transient network errors

  Authentication Options:

    -u, --username [arg]                       Username for authentication
    -p, --password [arg]                       Password for authentication
        --authenticationDatabase [arg]         User source (defaults to dbname)
        --authenticationMechanism [arg]        Authentication mechanism
        --awsIamSessionToken [arg]             AWS IAM Temporary Session Token ID

  TLS Options:

        --tls                                  Use TLS for all connections
        --tlsCertificateKeyFile [arg]          PEM certificate/key file for TLS
        --tlsCertificateKeyFilePassword [arg]  Password for key in PEM file for TLS
        --tlsCAFile [arg]                      Certificate Authority file for TLS
        --tlsAllowInvalidHostnames             Allow connections to servers with non-matching hostnames
        --tlsAllowInvalidCertificates          Allow connections to servers with invalid certificates
        --tlsCertificateSelector [arg]         TLS Certificate in system store
        --tlsDisabledProtocols [arg]           Comma separated list of TLS protocols to disable [TLS1_0,TLS1_1,TLS1_2]

  FLE Options:

        --awsAccessKeyId [arg]                 AWS Access Key for FLE Amazon KMS
        --awsSecretAccessKey [arg]             AWS Secret Key for FLE Amazon KMS
        --awsSessionToken [arg]                Optional AWS Session Token ID
        --keyVaultNamespace [arg]              database.collection to store encrypted FLE parameters
        --kmsURL [arg]                         Test parameter to override the URL of the KMS endpoint

  DB Address Examples:

        foo                                    Foo database on local machine
        192.168.0.5/foo                        Foo database on 192.168.0.5 machine
        192.168.0.5:9999/foo                   Foo database on 192.168.0.5 machine on port 9999
        mongodb://192.168.0.5:9999/foo         Connection string URI can also be used

  Examples:

        Start mongosh using 'ships' database on specified connection string:
        $ mongosh mongodb://192.168.0.5:9999/ships

  For more information on usage: https://docs.mongodb.com/mongodb-shell.

```

### CODE REVIEW

File: packages/i18n/src/locales/en_US.ts [hunk 1]
```diff
@@ -37,20 +38,20 @@ const translations: Catalog = {
...
-      dbAddressOptions: 'DB Address Examples',
+      kmsURL: 'Test parameter to override the URL of the KMS endpoint',
+      dbAddressOptions: 'DB Address Examples:',
       'dbAddress/foo': 'Foo database on local machine',
       'dbAddress/192/foo': 'Foo database on 192.168.0.5 machine',
       'dbAddress/192/host/foo': 'Foo database on 192.168.0.5 machine on port 9999',
       'dbAddress/connectionURI': 'Connection string URI can also be used',
-      fileNames: 'File Names',
+      fileNames: 'File Names:',
       filenameDescription: 'A list of files to run. Files must end in .js and will exit after unless --shell is specified.',
```
Comment (gribnoysup) [inline packages/i18n/src/locales/en_US.ts:53]: Those are not used anywhere anymore, right?

**Review by gribnoysup — APPROVED** [review]

**Comment by gribnoysup** [comment]
Oh, sorry, didn't notice the wip label

**Comment by rose-m** [comment]
No worries :D Yeah there are some in the translations that are not used right now but will be once we add them back. I'll leave them in there for now. The thing I noticed (hence the WIP) is that some tls options are missing.

**Comment by gribnoysup** [comment]
Ah, right, should've read them more carefully, makes sense!

**Pull request merged**